requirements.txt

```diff
@@ -24,7 +24,7 @@ pyasn1==0.4.3             # via pyasn1-modules, rsa
 python-dateutil==2.7.3    # via kubernetes
 pyyaml==3.12
 requests-oauthlib==1.0.0  # via kubernetes
-requests==2.19.1          # via kubernetes, requests-oauthlib
```

Comment:
nit: the `# via ...` is still valid. Neither kubernetes nor requests-oauthlib pins an explicit version of the library

Reply:
yeah, the `requirements.txt` is autogenerated via `pip-compile`, so if it sees the dep defined in `setup.py`, it assumes its a project dependency and doesn't add on the `# via ...` comment. I could add it back in, but it would just get removed whenever `requirements.txt` is regenerated. 

I'll make a note of this directly in `setup.py`.